docs/source/onnx_dynamo_onnxruntime_backend.rst

```diff
@@ -0,0 +1,10 @@
+ONNX Backend for TorchDynamo
+============================
+
+For a quick overview of ``torch.compiler``, see :ref:`torch.compiler_overview`.
+
+.. warning::
+  The ONNX backend for TorchDynamo is under active development and is
+  subject to rapid change.
+
+.. autofunction:: torch.onnx.is_onnxrt_backend_supported
```

Comment:
Would be good to replicate the same demo you have for the exporter and invoke it via DORT here. Also mention that `onnxruntime` needs to be installed in addition to `onnx` and `onnxscript`.

This can be a separate PR though.

Reply:
it is best to separate DORT and onnx exporter in separate documentation.
They should not be confused.